testplan/web_ui/testing/src/AssertionPane/Assertion.js

```diff
@@ -20,9 +20,12 @@ import NotImplementedAssertion from './AssertionTypes/NotImplementedAssertion';
 import AssertionHeader from './AssertionHeader';
 import AssertionGroup from './AssertionGroup';
 import {BASIC_ASSERTION_TYPES} from '../Common/defaults';
-import XYGraphAssertion from './AssertionTypes/GraphAssertions/XYGraphAssertion';
-import DiscreteChartAssertion from './AssertionTypes/GraphAssertions/DiscreteChartAssertion';
+import XYGraphAssertion
+  from './AssertionTypes/GraphAssertions/XYGraphAssertion';
+import DiscreteChartAssertion
+  from './AssertionTypes/GraphAssertions/DiscreteChartAssertion';
```

Comment:
is it feasible to hoist the exports like:

DiscreteChartAssertion.js => GraphAssertions/index.js => AssertionTypes.js 

so we can just do `import DiscreteChartAssertion from './AssertionTypes';` ?

Reply:
I was only adding in line-breaks here to fix new line-length linting rules added by the new react-scripts version.

Hoisting the exports sounds reasonable though.